### PR TITLE
Adds and updates key for trained questions in relevancy search

### PR DESCRIPTION
**Pull request by Zachary-Dernar**

### CODE REVIEW

**Comment by Zachary-Dernar** [comment]
@maniax89 my concerns are with the vertical spacing to contain the key and view-all button. when I changed the new `.question_bar--bottom` to `float: right` the elements, it collapsed the background and overlapped those elements into the footer, so I added padding from the container above it to contain it.

File: client/knowledge_base_search/src/containers/QuestionBarContainer/styles.scss [hunk 1]
```diff
@@ -10,14 +10,21 @@
     display: flex;
   }
 
+  &--bottom {
+    margin-top: 0.5rem;
+    display: flex;
+    float: right;
```
Comment (maniax89) [inline client/knowledge_base_search/src/containers/QuestionBarContainer/styles.scss]: floats are the old way to do what flexbox does, so you probably shouldn't have this anywhere when using flexbox. instead look up how to align the content https://css-tricks.com/snippets/css/a-guide-to-flexbox/#article-header-id-6

File: client/knowledge_base_search/src/containers/SearchContainer/styles.scss [hunk 2]
```diff
@@ -13,13 +13,31 @@
...
+
+    &--text {
+      color: color('teal', 80);
+      font-size: 0.75rem;
+      font-weight: 300;
+      margin-right: 1rem;
+    }
+  }
+
+  .view_all_button--span {
```
Comment (maniax89) [inline client/knowledge_base_search/src/containers/SearchContainer/styles.scss]: remove this

File: client/knowledge_base_search/src/containers/SearchContainer/styles.scss [hunk 3]
```diff
@@ -13,13 +13,31 @@
...
+      margin-right: 0.5rem;
+      border: 1px solid color('white', 10);
+      border-radius: 50%;
+      background-color: color('purple', 50);
+    }
+
+    &--text {
+      color: color('teal', 80);
+      font-size: 0.75rem;
+      font-weight: 300;
```
Comment (maniax89) [inline client/knowledge_base_search/src/containers/SearchContainer/styles.scss]: maybe see if you can use the typography settings defined in https://watson-developer-cloud.github.io/ui-components/style.html#typography

```
font-size: get('weight light');
```
maybe?

**Review by maniax89 — APPROVED** [review]

**Pull request merged**